NERODevelopment/content/DetailDisplay.qml

```diff
@@ -0,0 +1,78 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.0
+RowLayout {
+    id: rowLayout
+    anchors.fill: parent
+    property string shutdownFlowTask: "Task" //controller.task
+    property string status: "null" //controller.status
+    property bool ok: false //controller.ok
+    property bool clear: false //controller.clear
+    property bool taskHovered: true
+    property bool highlight: false //controller.highlight
+    width: 200
+    height:50
+
+
+    Button {
```

Comment:
this doesnt need to be a button. theres no user input on the gui, we get button clicks from the model.